surfinpy/bulk_mu_vs_t.py

```diff
@@ -115,9 +114,11 @@ def evaluate_phases(data, bulk, x, y,
     S = np.array([])
     new_data_svib = 0
     if bulk.entropy:
-        new_data_svib = ut.build_entgrid(bulk.svib, x, ynew)            
+        new_bulk_svib = ut.build_zgrid(bulk.avib, x)       
 
     for k in range(0, nphases):
```

Comment:
This does not make sense and causes errors. 
new_data_svib is passed to calculate_bulk_energy and the code fails because it is not allowed to be False. 
As above, you need to provide some sort of hardcoded value that is defaulted to if data[k].entroy is False

Reply:
Within the old logic thsi would be solved by a line like......

```suggestion
    for k in range(0, nphases):
            new_data_svib = 0
        if data[k].entropy:
            new_data_svib = ut.build_zgrid(data[k].avib, x)
```

Reply:
Changed this originally as noticed that the entropy term was only being calculated and applied to the phase (data[k]) (new_data_svib) and not considering the entrropy of the bulk phase.  Created a new variable new_bulk_svib and checked if the user wants to calculate entropy for the bulk phase.